internal/models/speaker.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+type CategoryType string
+
+const (
+	CategoryTypeIndustry    CategoryType = "industry"
+	CategoryTypeAcademic    CategoryType = "research"
+	CategoryTypeStudent     CategoryType = "student"
+	CategoryTypeBoardMember CategoryType = "board member"
+)
+
+type Speaker struct {
+	ID       uint         `gorm:"primarykey" json:"id"`
+	Name     string       `gorm:"not null" json:"name"`
+	Title    string       `gorm:"not null" json:"title"`
+	Cateogry CategoryType `gorm:"not null" json:"category"`
+	Events   []Event      `gorm:"many2many:speaker_events;" json:"events"`
```

Comment:
Can we make this optional? When migrating to our new website, we might not migrate all old events. Hence, it would be nice to be able to showcase some of our previous cool speakers without actually having to re-add the events they were associated with.